plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProjectTest.java

```diff
@@ -12,6 +22,27 @@
 
   @Mock private IAdaptable adaptable;
 
+  private SubMonitor monitor = SubMonitor.convert(new NullProgressMonitor());
```

Comment:
I think you could just make this `IProgressMonitor monitor = new NullProgressMonitor()` — the SubMonitor doesn't actually add anything.